Dockerfile

```
@@ -23,4 +23,7 @@ COPY poetry.lock pyproject.toml ./
 
 RUN poetry install
 
-COPY . .
\ No newline at end of file
+COPY . .
+
+ENTRYPOINT [ "poetry", "run", "kedro", "run" ]

```

```suggestion
ENTRYPOINT [ "kedro", "run" ]
```
Ponieważ mamy już `POETRY_VIRTUALENVS_CREATE=false` to wszystko instaluje się w głównym środowisku, więc nie trzeba przez poetry.